kubernetes/base/rbac.yaml

```diff
@@ -25,7 +24,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: descheduler-sa
-  namespace: kube-system
```

Comment:
ServiceAccount is namespaced, `namespace` field has to stay as it is.

`namespace` of `ClusterRoleBinding` right below can drop the field

Reply:
@ingvagabund Fixed it. Thank you!